### ISSUE TITLE
Suggestion

### ISSUE THREAD

**Issue by N1NJ4smurf** [denno020/linkslist#18]
G'day... love that you've created this, and love that it's free... thanks mate!

Two suggestions:
- would be awesome if we could click-drag to reorder the links
- would be great if we could edit the link to give it a title and even change/remove the preview image

**Comment by denno020** [comment]
Hey @N1NJ4smurf, you're welcome!

As for the feature requests, they're on my to do list and I'm in the process of implementing them already :)..

I'm still not sure if I'm going to make these features available for free or part of the premium subscription though.

I'll be sure to let you know when the features are released either way!

**Comment by N1NJ4smurf** [comment]
Yeah, fair enough. Depending on cost it's something I'd consider subscribing to 👍🏼

**Comment by denno020** [comment]
That's great to hear :). 
There's an expression of interest form on the landing page that you can sign up to in order to be notified when the premium features are ready. I plan on doing a deal for people who have expressed interest and giving them the option to sign up for a better price. When the app is updated, you'll receive an email and then be able to make up your mind if you want to sign up or not.
If you haven't already, scroll down to the Pricing section of the landing page to enter your email :)

**Comment by N1NJ4smurf** [comment]
Too easy 😎
